WindowsServerDocs/troubleshoot/windows-server-support-solutions.md

```diff
@@ -1,183 +1,157 @@
-<hr>
-                        <p><a href="../identity/ad-fs/operations/manage-ssl-certificates-ad-fs-wap-2016.md">Managing SSL Certificates in AD FS and WAP in Windows Server 2016</a></p>
-<hr>
-                        <p><a href="../identity/ad-fs/operations/access-control-policies-in-ad-fs.md">Access Control Policies in Windows Server 2016 AD FS</a></p>
-                      </div>
-                 </div>
-            </div>
-        </div>
-    </li>
-<li>
- <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="../media/i-troubleshoot.svg" alt="" />
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>Active Directory replication</h3>
-<hr> 
-                         <a href="../identity/ad-ds/manage/troubleshoot/troubleshooting-active-directory-replication-problems.md">Troubleshooting Active Directory Replication Problems</a>
-<hr>
-                         <a href="https://www.microsoft.com/en-in/download/details.aspx?id=30005">e2e: How to troubleshoot common Active Directory replication errors</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/3108513">Download Active Directory Replication Status Tool from the Microsoft Download Center</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/2028495">Troubleshooting AD Replication error 8606: Insufficient attributes were given to create an object</a></p>
-<hr>
-                         <a href="https://support.microsoft.com/kb/837932">Event ID 2108 and Event ID 1084 occur during inbound replication of Active Directory in Windows 2000 Server and in Windows Server 2003</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/2645996">Troubleshooting AD Replication error 8451: The replication operation encountered a database error</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/2025726">Troubleshooting AD Replication error 1127: While accessing the hard disk, a disk operation failed even after retries</a>
-<hr>
-                         <a href="https://technet.microsoft.com/library/cc816907.aspx">Clean Up Server Metadata</a>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-</ul>
\ No newline at end of file
+# Troubleshooting Windows Server components
+
+>- [!TIP]
+> Looking for information about older versions of Windows Server? Check out our other - [Windows Server libraries](/previous-versions/windows/) on docs.microsoft.com. You can also - [search this site](https://docs.microsoft.com/search/index?search=Windows+Server&dataSource=previousVersions) for specific information.
```

Comment:
I notice you have added dashes within this text row, presumably to create bullet points for the links. MarkDown may not translate them as such, unless they are the first character of a line. Consider adding HTML breaks ( `<br>` ) before those dashes to make them work as bullet points, or add actual line breaks with their own MD quote markers to get the extra lines in the Tip Note. Example:
```
> Looking for information about older versions of Windows Server? Check out our other
> - [Windows Server libraries](/previous-versions/windows/) on docs.microsoft.com. You can also
> - [search this site](https://docs.microsoft.com/search/index?search=Windows+Server&dataSource=previousVersions) for specific information.
```